api.planx.uk/modules/send/bops/bops.ts

```diff
@@ -178,21 +180,14 @@ const sendToBOPSV2 = async (
   // a local or staging API instance should send to the BOPS staging endpoint
   // production should send to the BOPS production endpoint
   const localAuthority = req.params.localAuthority;
-  const env = process.env.NODE_ENV === "production" ? "production" : "staging";
+  const env =
+    process.env.APP_ENVIRONMENT === "production" ? "production" : "staging";
   const bopsSubmissionURL = await $api.team.getBopsSubmissionURL(
     localAuthority,
     env,
   );
+  console.log({ bopsSubmissionURL });
```

Comment:
nit: is it intentional to keep this `console.log()` and the one above on line 53, or were these temporary debugging measures?

Reply:
Ha good catch sorry! Yep will tidy this up.